src/Prefix.jl

```diff
@@ -341,10 +346,12 @@ function setup_workspace(build_path::AbstractString, sources::Vector;
 
     # We now set up two directories, one as a source dir, one as a dest dir
     srcdir = joinpath(workspace, "srcdir")
-    destdir = joinpath(workspace, "destdir")
+    target_destdir = destdir(workspace, target_platform)
+    host_destdir = destdir(workspace, host_platform)
     metadir = joinpath(workspace, "metadir")
-    wrapperdir = joinpath(workspace, "compiler_wrappers")
-    mkdir.((srcdir, destdir, metadir))
+    mkpath.((srcdir, target_destdir, host_destdir, metadir))
+    # Create the symlink /workspace/destdir -> /workspace/TARGET_TRIPLET/destdir
```

Comment:
```suggestion
    # Create the symlink /workspace/destdir -> /workspace/TARGET_TRIPLET/destdir
    # Necessary for compatibility with recipes that hardcode `/workspace/destdir` in them,
    # as well as `.pc` files that contain absolute paths to `/workspace/destdir/...`
```

As a side-note, I just learned about the `${pcfiledir}` variable in `.pc` files; you can use that to set up properly-relocatable `.pc` files by setting things like:

```
prefix=${pcfiledir}/../..
libdir=${prefix}/lib
...
```

So we should probably write an Audit pass to do that; would solve a lot of our `.pc` problems.  Issue here: https://github.com/JuliaPackaging/BinaryBuilderBase.jl/issues/108